draft-fossati-tls-attestation.md

```diff
@@ -652,6 +622,10 @@ the attester.
 ~~~~
 {: #figure-extension-results title="TLS Extension Structure for Attestation Results."}
 
+In the Passport model, both attestation results and an indication of key attestation are
```

Comment:
The indication of key attestation is part of the attestation results.

Reply:
I'll keep the text, because the next sentence says we have not defined it.